funcx_sdk/funcx/sdk/utils/printing.py

```diff
@@ -36,7 +36,7 @@ def print_table(
 
     for row in table_rows:
         if len(row) < max_columns:
-            table.add_row(row + ["" for i in range(max_columns - len(row))])
+            table.add_row(row + [""] * (max_columns - len(row)))
         else:
             table.add_row(row)
```

Comment:
If you use this trick, there's no need for the if-condition.  The if-condition is effectively built into the `max_columns - len(row)` logic.